frontend/src/Pages/AdminPanel.tsx

```diff
@@ -152,95 +139,80 @@ export const AdminPanel = ({
           <Tbody>
             {groups.map(({ name: group, workStations }, i) => {
               return workStations.map((workStation, j) => {
-                const id = (i * j + j) * 5;
+                const id = i * j + j;
```

Comment:
NIT: Could be cleaner

Reply:
Cleaned up in https://github.com/gpuctl/gpuctl/pull/140/commits/5470130f6c9b081e4fdbfc4f7a18df0b1eb5fd71